docs/changelog.rst

```diff
@@ -1,5 +1,9 @@
 Changelog
 =========
+in progress
```

Comment:
3.10 with todays date please

Reply:
Also please ensure the CI passes.

Reply:
> 3.10 with todays date please

I don't know your plan for the version number and date of the next release, so I don't think I should write those. Some ppl call this section "[Unreleased]", that's the same idea

https://keepachangelog.com/en/1.0.0/

Reply:
> Also please ensure the CI passes.

Now flake8 check passes

Reply:
Also just pick the next bugfix version instead of unreleased/in-progress

Reply:
sorry, it's your project. I don't want to set version number for you.

Reply:
I'm asking you to do it though 😉

Reply:
done